### PR TITLE
feat: abstract VerificationService as one class

### PR DESCRIPTION
**Pull request by drshrey**

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -4,5 +4,9 @@ mock>=1.0.1
 flake8>=2.1.0
 tox>=1.7.0
 twilio==5.4.0
+nose==1.3.7
+djangorestframework==3.3.3
+django-environ==0.4.0
+django-phonenumber-field
```
Comment (glemmaPaul) [inline requirements-test.txt]: add version, you could see the current version with `pip freeze`

File: tests/test_services.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        """
+        (self.service.send_verification(number=self.phone_number, request=self.request))
+
+    @raises(ValueError)
+    def test_send_verification_with_no_number(self):
+        """ Test that send_verification with no number specified
+            raises a ValueError
+        """
+        self.service.send_verification('', self.request)
+        ok_(self.mock.called)
```
Comment (glemmaPaul) [inline tests/test_services.py]: Is this ok_ ever being called when @raises is ion there?

File: tests/test_services.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        factory = APIRequestFactory()
+        self.request = factory.get('/')
+        self.mock = mock
+        self.phone_number = '+13478379634'
+
+    def test_send_verification_with_valid_number_and_request(self):
+        """ Test that send_verification with a valid number and request,
+            is successful
+        """
+        (self.service.send_verification(number=self.phone_number, request=self.request))
```
Comment (glemmaPaul) [inline tests/test_services.py]: Should the ok_ beneath this test not be in here?

File: requirements-test.txt (hunk 1 above)
Comment (drshrey) [inline requirements-test.txt]: sounds good

File: tests/test_services.py (hunk 2 above)
Comment (drshrey) [inline tests/test_services.py]: i should get rid of this. not necessary

File: tests/test_services.py (hunk 3 above)
Comment (drshrey) [inline tests/test_services.py]: fixed

**Comment by glemmaPaul** [comment]
LGTM

**Pull request merged**